pkg/clients/azure/aks.go

```diff
@@ -30,7 +30,7 @@ import (
 
 const (
 	// AgentPoolProfileNameFmt is a format string for the name of the automatically created cluster agent pool profile
-	AgentPoolProfileNameFmt = "%s-nodepool"
+	AgentPoolProfileName = "agentpool"
```

Comment:
Did you already confirm that agent pool profile names do not have to be unique? i.e. duplicate names in the same resource group are OK?

Reply:
yes, I did. I created multiple clusters with different agentpool settings (instance types).